tests/basic/storage_pool_test.go

```diff
@@ -9904,3 +9904,255 @@ var _ = Describe("{AddDriveWithKernelPanic}", func() {
 	})
 
 })
+
+func selectPoolDeletableNode () (*node.Node){
```

Comment:
why only check for kvdb nodes? What about metadata / journal on the pool that we are selecting?

Reply:
because internal kvdb nodes cannot become storageless node. we cannot delete pool on this node if there is only one. So, if all nodes are internal kvdb nodes, we have to add a pool to run test.

Reply:
If auto journal is enabled on the node, even then we can't delete all the pools. That's also a good check to add so that if the job is ever run with a different setup we'll know exactly why it fails or we can even skip the test if no suitable node is found.